.travis.yml

```
@@ -117,7 +117,7 @@ deploy:
     bucket_name: wire-webapp-builds
     on:
       tags: true
-      condition: $TRAVIS_TAG =~ .*-staging.* AND $DISTRIBUTION = wire
+      condition: $TRAVIS_TAG =~ .*-staging.* && $DISTRIBUTION = wire
```

Shouldn't it be `==`?

```suggestion
      condition: $TRAVIS_TAG =~ .*-staging.* && $DISTRIBUTION == "wire"
```

---

I haven't figured out all the details but it seems to be a Travis thing here: https://docs.travis-ci.com/user/conditions-v1